package.json

```diff
@@ -45,5 +45,8 @@
     "test": "eslint Gruntfile.js lib test/test.js && npm run nodeunit",
     "test-0.10": "npm run nodeunit",
     "nodeunit": "nodeunit test/test.js"
+  },
+  "dependencies": {
```

Comment:
I feel bad about this, but I don't know what else to do about it.

Reply:
I mean we could just hardcode the names?

Reply:
Yeah, true; I guess there isn't much likelihood that they'll change in the future.

Reply:
Yeah, we don't use that dependency in Sanctuary. Let's see what @vqvu thinks.

Reply:
+1 to just hard-coding the names, but also keep fantasy-land as a `devDepedencies` and using it in our tests. That way we can verify our names match the spec without having to actually include the spec as a real dependency.